.changeset/fifty-pandas-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@builder.io/sdk': major
+---
+
+Fix: Reintroduced `JSON.stringify(userAttributes)` change to standardize parsing logic and preserve strings. This is a breaking change as it doesn't require manual stringification of `userAttributes` values. Ensure that attributes are not manually stringified before passing them to avoid potential issues.
```

Comment:
we should include a a release note in the React SDK that explains why it's being bumped a major version too.